com.unity.render-pipelines.universal/Shaders/XR/XROcclusionMesh.shader

```diff
@@ -51,7 +51,6 @@ Shader "Hidden/Universal Render Pipeline/XR/XROcclusionMesh"
         Pass
         {
             ZWrite On ZTest Always Blend Off Cull Off
-            ColorMask 0
```

Comment:
Removing colorMask 0 from occlusion shade can be expensive since the shader used to be full-screen depth-only now writes color as well. Again, we should defer to the effect (bloom) to decide what color should be cleared (i.e. in XR mode, clear the first bloom buffer).